### PR TITLE
TiKV configuration: fixed some default configurations

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)

- fixed some default configurations for RocksDB

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs/pull/5969
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1181,6 +1186,22 @@ rocksdb lockcf 相关配置项。
...
 
+### `whole-key-filtering`
+
++ 开启将整个 key 放到 bloom filter 中的开关。
++ 默认值：`true`
+
+### `compression-per-level`
+
++ 每一层默认压缩算法，默认：前两层为 No，后面 5 层为 lz4。
++ 默认值：["no", "no", "no", "no", "no", "no", "no"]
```
Comment (yiwu-arbug) [inline tikv-configuration-file.md:1197]: lockcf 每一层的默认压缩算法都是 no，所以默认值描述是对的，但是上面的文字描述不正确。

Comment (en-jin19) [reply tikv-configuration-file.md:1197]: 如果在文字描述部分只保留 “每一层默认压缩算法” 这段文字是否会可以呢 ? @yiwu-arbug

Comment (yiwu-arbug) [reply tikv-configuration-file.md:1197]: 可以

**Review by yiwu-arbug — APPROVED** [review]

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -900,10 +900,10 @@ rocksdb defaultcf 相关的配置项。
 + 开启优化 filter 的命中率的开关。
 + 默认值：true
 
-### `whole_key_filtering`
+### `whole-key-filtering`
 
 + 开启将整个 key 放到 bloom filter 中的开关。
```
Comment (TomShawn) [inline tikv-configuration-file.md:905]: 请和英文保持一致

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1154,6 +1154,11 @@ rocksdb writecf 相关的配置项。
 + 设置 compaction guard 的启用状态。compaction guard 优化通过使用 TiKV Region 边界分割 SST 文件，帮助降低 compaction I/O，让 TiKV 能够输出较大的 SST 文件，并且在迁移 Region 时及时清理过期数据。
 + 默认值：true
 
+### `compression-per-level`
+
++ 每一层默认压缩算法，默认：前两层为 No，后面 5 层为 lz4。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1159]: 和英文保持一致

**Review by TomShawn — APPROVED** [review]

**Pull request merged**